migration/serversideencryption.php

```diff
@@ -83,19 +84,27 @@ public function run(IOutput $output) {
 		}
 	}
 
-	private function fetchAll($sql){
-		return $this->db->executeQuery($sql)->fetchAll();
+	private function fetchAll(string $table) {
+		// restrict access to passman tables
+		if (substr($table, 0, strlen('passman_')) === 'passman_') {
```

Comment:
What is with this IF?

Reply:
its not really needed. Thought it would be a security improvement if fetchAll() is restricted to passman tables.
But its private so ... hmm dont care about it :)
Removed it